dashboard/lib/dynamic_config/adapters/dynamodb_adapter.rb

```diff
@@ -0,0 +1,64 @@
+# A DynamoDB adapter class that allows us to use
+# dynamodb as a persistent store with the datastore_cache
+require 'aws-sdk'
+require 'oj'
+
+class DynamoDBAdapter
+  # @param table_name [String] the name of the dynamodb table to use
+  def initialize(table_name)
+    @table_name = table_name
+    @client = Aws::DynamoDB::Client.new
+  end
+
+  # @param key [String]
+  # @returns [String]
+  def get(key)
+    resp = @client.get_item({
+      table_name: @table_name,
+      key: {
+        'data-key' => key
+      }
+    })
+    return nil if resp.item.nil?
+    Oj.load(resp.item['data-value'])
```

Comment:
I believe this throws an exception if the item can't be parsed.  Is that the desired behavior?  Please either document that this method can throw and handle it in the caller, or handle it here, log an error, and return nil.

I am concerned we might break all of our servers at once if we ever had a bug that allowed invalid JSON to be upload (e.g. something not allowed by the OJ's stricter behavior.)